### PR TITLE
Updating Automation's resource schemas

### PR DESCRIPTION
**Pull request by vrdmr**

### CODE REVIEW

**Comment by vrdmr** [comment]
Hi @Tiano2017, Could you please take a look at this PR. Thanks.

File: schemas/2015-10-31/Microsoft.Automation.json [hunk 1]
```diff
@@ -740,6 +744,66 @@
       ],
       "description": "Microsoft.Automation/automationAccounts/variables"
     },
+    "automationAccounts_watchers": {
```
Comment (TianoMS) [inline schemas/2015-10-31/Microsoft.Automation.json]: >automationAccounts_watchers [](start = 5, length = 27)

add the new type in deploymentTemplate.json?

Comment (vrdmr) [reply schemas/2015-10-31/Microsoft.Automation.json]: @TianoMS Added to deploymentTemplate.json and the tests for it.

**Comment by vrdmr** [comment]
Ping. @TianoMS

File: schemas/2017-05-15-preview/Microsoft.Automation.json [hunk 2]
```diff
@@ -701,6 +697,7 @@
         },
         "duration": {
           "type": "string",
+          "format": "duration",
```
Comment (TianoMS) [inline schemas/2017-05-15-preview/Microsoft.Automation.json]: >"format": "duration", [](start = 10, length = 21)

during format is not supported by Visual Studio. let's remove it for now..

**Comment by Tiano2017** [comment]
PR looks good. Just one small issue. please take a look.

**Comment by vrdmr** [comment]
Ping. @TianoMS

**Pull request merged**